apps/api/src/services/library-traffic.ts

```diff
@@ -0,0 +1,35 @@
+import type { libraryTrafficQuerySchema } from "$schema";
+import type { database } from "@packages/db";
+import { and, eq } from "@packages/db/drizzle";
+import { libraryTraffic } from "@packages/db/schema";
+import type { z } from "zod";
+
+type LibraryTrafficServiceInput = z.infer<typeof libraryTrafficQuerySchema>;
+
+export class LibraryTrafficService {
+  constructor(private readonly db: ReturnType<typeof database>) {}
+
+  getLibraryTraffic(input: LibraryTrafficServiceInput) {
+    const queryParams = [];
```

Comment:
We typically call an array used for this kind of pattern `conds`. Let me know if there's a compelling reason for this alternative.